webrecorder/webrecorder/rec/main.py

```diff
@@ -1,12 +1,15 @@
 from gevent import monkey; monkey.patch_all()
 
+import os
+from ast import literal_eval
+
 from webrecorder.utils import load_wr_config, init_logging, spawn_once
```

Comment:
```suggestion
from webrecorder.utils import load_wr_config, init_logging, spawn_once, get_bool
```
To use get_bool below